qml/FileListPage.qml

```diff
@@ -146,6 +147,12 @@ Page {
                                            : qsTrId("sailfish-office-me-hide_search")
                 onClicked: page.searchEnabled = !page.searchEnabled
             }
+            
+            MenuItem {
+                //% "Sort"
+                text: qsTrId("sailfish-office-me-sort")
+                onClicked: pageStack.push("SortTypeSelectionDialog.qml", {filterModel: filteredModel})
```

Comment:
Could use animatorPush()

Reply:
Why is  it better to use instead of push()? Is there any issue in documentation about this method?

Reply:
Starts push already before the page is created so the UI feels faster. Not documented at the moment due to being still slightly work in progress.

Reply:
Undocumented method usage is a bad way, IMHO. Because Lord knows what will happen to it further, how it will be change or whether it will exist

Reply:
Besides it's already used on nearly all other places here. One instance of push() but i recall there was some specific detail related to it.